plasmapy/formulary/collisions.py

```diff
@@ -2033,3 +2038,178 @@ def coupling_parameter(
+    ):
+        r"""
+        Compute collision frequencies between test particles (labeled 'a') and field particles (labeled 'b')
+
+        Parameters
+        ----------
+        test_particle : |Particle|
+            The test particle streaming through a background of field particles.
+
+        field_particle : |Particle|
+            The background particle being interacted with.
+
+        v_a : `~astropy.units.Quantity`, optional
+            The relative velocity between particles. If not provided, T_a must be specified and
+            thermal velocity is assumed: :math:`μ v_a^2 \sim 2 k_B T_a` where
+            :math:`μ` is the reduced mass.
+
+        T_a : `~astropy.units.Quantity`, optional
+            The temperature of the test species. Only necessary if :math:`v_a` is not provided.
+
+        n_b : `~astropy.units.Quantity`
+            The number density of the background field particles.
+
+        T_b : `~astropy.units.Quantity`
+            The temperature of the background field particles.
+
+        Coulomb_log : `~astropy.units.Quantity`
+            The value of the Coulomb logarithm evaluated for the two interacting particles.
+
+        Raises
+        ------
+        `ValueError`
+            If both :math:`v_a` and :math:`T_a` are specified, or neither
+            :math:`v_a` nor :math:`T_a` are specified.
+
+        Notes
+        -----
+        The collision frequencies between two interacting particles are given by four differential equations:
+
+            slowing down - :math:`\frac{d\textbf{v}_{α}}{dt}=-ν_{s}^{α\backslashβ}\textbf{v}_{α}`
+
+            transverse diffusion - :math:`\frac{d}{dt}\left(\overline{\textbf{v}}_{α}-\overline{\textbf{v}}_{α}\right)_{⊥}^{2}=ν_{⊥}^{α\backslashβ}v_{α}^{2}`
+
+            parallel diffusion - :math:`\frac{d}{dt}\left(\overline{\textbf{v}}_{α}-\overline{\textbf{v}}_{α}\right)_{∥}^{2}=ν_{∥}^{α\backslashβ}v_{α}^{2}`
+
+            energy loss - :math:`\frac{d}{dt}v_{α}^{2}=-ν_{ϵ}^{α\backslashβ}v_{α}^{2}`
+
+        For general solutions and limiting cases we encourage the curious reader to refer to p. 31 of :cite:t:`nrlformulary:2019`
+
+        This function uses CGS units under the hood to coincide with our references, but passing MKS units is acceptable.
+
+        Examples
+        --------
+        >>> import astropy.units as u
+        >>> T_a = 1 * u.eV
+        >>> n_b = 1e20 * u.cm**-3
+        >>> T_b = 1e3 * u.eV
+        >>> Coulomb_log = 10 * u.dimensionless_unscaled
+        >>> frequencies = CollisionFrequencies(
+        ...     "e-", "e-", T_a=T_a, n_b=n_b, T_b=T_b, Coulomb_log=Coulomb_log
+        ... )
+        >>> frequencies.slowing_down
+        <Quantity 1.83701432e+11 Hz>
+
+        See Also
+        --------
+        ~plasmapy.formulary.collisions.Coulomb_logarithm : Evaluates the Coulomb
+            logarithm for two interacting electron species.
+
+        ~plasmapy.formulary.collisions.collision_frequency : The Lorentz collision frequency.
```

Comment:
Remove this from here (since we're deprecating that function) but put a See Also reference to this new class in the collision_frequency docstring.